### PR TITLE
Cherry-pick #9581 into release/0.43

### PR DESCRIPTION
**Pull request by lukelee-sl**
**Description**:
Increase the number of KV pairs allowed for contract storage by a factor of 100 x

### CODE REVIEW

File: hedera-node/configuration/mainnet/bootstrap.properties [hunk 1]
```diff
@@ -19,3 +19,4 @@ accounts.maxNumber=20_000_000
 #Overrides that differ based on the network
 ledger.id=0x00
 contracts.chainId=295
+contracts.maxKvPairs.individual=16384000
```
Comment (netopyr) [inline hedera-node/configuration/mainnet/bootstrap.properties:22]: I don't think we need to change the override files. Updating the defaults is sufficient.

**Review by netopyr — DISMISSED** [review]

**Review by Nana-EC — APPROVED** [review]

**Review by povolev15 — APPROVED** [review]

File: hedera-node/hedera-app/src/test/resources/bootstrap.properties [hunk 2]
```diff
@@ -79,8 +79,8 @@ contracts.keys.legacyActivations=1058134by[1062784]
 contracts.knownBlockHash=
 contracts.localCall.estRetBytes=32
 contracts.maxGasPerSec=15000000
-contracts.maxKvPairs.aggregate=500_000_000
-contracts.maxKvPairs.individual=163_840
+contracts.maxKvPairs.aggregate=500000000
```
Comment (poulok) [inline hedera-node/hedera-app/src/test/resources/bootstrap.properties:82]: why remove the underscores in this value? They make it easier to read and verify that the correct value is set

File: hedera-node/hedera-app/src/test/resources/bootstrap.properties [hunk 3]
```diff
@@ -79,8 +79,8 @@ contracts.keys.legacyActivations=1058134by[1062784]
...
+contracts.maxKvPairs.individual=16384000
```
Comment (poulok) [inline hedera-node/hedera-app/src/test/resources/bootstrap.properties:83]: Please add underscores

File: hedera-node/hedera-app/src/test/resources/bootstrap.properties (hunk 2 above)
Comment (lukelee-sl) [reply hedera-node/hedera-app/src/test/resources/bootstrap.properties:82]: The underscores were causing some of the HAPI tests to fail in the modularized code with `NumberFormatException`. Apparently, the logic for parsing the config values with the underscores has not been implemented in some cases (most likely the new config system)? We probably went overboard on removing them but we wanted to be sure that the tests would pass.

**Review by rbair23 — CHANGES_REQUESTED** [review]

**Review by rbair23 — APPROVED** [review]

File: hedera-node/hedera-app/src/test/resources/bootstrap.properties (hunk 2 above)
[the remark this answers is not in the document]
Comment (lukelee-sl) [reply hedera-node/hedera-app/src/test/resources/bootstrap.properties]: Restored the original value `500_000_000`

Comment (lukelee-sl) [reply hedera-node/hedera-app/src/test/resources/bootstrap.properties]: > Can you file an issue for Hendrik to add underscore support to number properties in the new config system? I agree, without underscores, we're likely to make a mistake.

Will do.

Comment (lukelee-sl) [reply hedera-node/hedera-app/src/test/resources/bootstrap.properties]: an issue was already created by @david-bakin-sl 
https://github.com/hashgraph/hedera-services/issues/9589

**Review by Nana-EC — APPROVED** [review]

**Review by tinker-michaelj — APPROVED** [review]
LGTM

**Review by povolev15 — APPROVED** [review]
LGTM

File: hedera-node/hedera-app/src/test/resources/bootstrap.properties (hunk 2 above)
[the remark this answers is not in the document]
Comment (Nana-EC) [reply hedera-node/hedera-app/src/test/resources/bootstrap.properties]: Under scores restored

File: hedera-node/hedera-app/src/test/resources/bootstrap.properties (hunk 3 above)
Comment (Nana-EC) [reply hedera-node/hedera-app/src/test/resources/bootstrap.properties]: Under scores restored

File: hedera-node/hedera-app/src/test/resources/bootstrap.properties (hunk 2 above)
Comment (Nana-EC) [reply hedera-node/hedera-app/src/test/resources/bootstrap.properties]: Under scores restored and ticket opened

**Pull request merged**